README.md

```diff
@@ -20,24 +23,32 @@ from datetime import datetime
 # make a pv site object
 site = PVSite(latitude=51.75, longitude=-1.25, capacity_kwp=1.25)
 
-# run model for today, using ICON NWP data
+# run model for today
 predictions_df = run_forecast(site=site, ts=datetime.today(), nwp_source="icon")
 ```
 
-which should result in a time series similar to this one: 
+which should result in a time series similar to this one:
 
 ![https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/blob/main/predictions.png?raw=true](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/blob/main/predictions.png?raw=true)
 
+A colab notebook providing some examples can be found [here](https://colab.research.google.com/drive/1qKDFRpq4Hk-LHgWuDsz_Najc3Zq-GVNY?usp=sharing).
```

Comment:
I havent looked at the notebook, is it worth committing the note book in this repo? I guess the advantage of it being on colab is anyone can use it very easily

Reply:
I think a link to a Google collab notebook and uploading the notebook to the examples folder would be good. Its quite similar to the example.ipynb, so maybe the new notebook can replace the example.ipynb version. (theres a small bug with the example.ipynh due to my environment I had when i made it so happy for it to be replaced!